WalletWasabi.Backend/wwwroot/faq.html

```diff
@@ -0,0 +1,400 @@
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link" href="index.html">The Wallet</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="contribution.html">Contribution</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="about.html">About us</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://blog.wasabiwallet.io" target="_blank">Blog</a>
+                    </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="faq.html">Help</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="btn btn-secondary" href="index.html#download">Download</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="fixed-img">
+        <img src="img/04_support.png" alt="Wasabi Wallet - Help">
+    </div>
+
+    <div class="content">
+        <section class="blured-section p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8 offset-md-2">
+                        <h1 class="text-center title-lined mb-5">FAQ</h1>
+
+                        <div class="card">
+                            <input type="checkbox" id="faq1">
+                            <label for="faq1">
+                                WHAT IS A “coinjoin”?
+                            </label>
+                            <div class="card-content">
+                                <p>Coinjoin is a mechanism by which multiple participants combine their coins (or UTXOs) into one large transaction with multiple inputs and multiple outputs. An observer cannot determine which output belongs to which input, and neither can the participants themselves. This makes it difficult for outside parties to trace where a particular coin originated from and where it was sent to (as opposed to regular bitcoin transactions, where there is usually one sender and one receiver).</p>
+
+                                <p>This can be done with non-custodial software like Wasabi Wallet that eliminates the risk of funds disappearing or being stolen. Each of the signatures are created on the participants’ computers, so anyone trying to connect the signatures is unable to alter the transaction or redirect the funds. The funds will always be in a Bitcoin address that you control. </p>
```

Comment:
```suggestion
                                <p>This can be done with non-custodial software like Wasabi Wallet that eliminates the risk of funds disappearing or being stolen. Each of the signatures are created on the participants’ computers, and a transaction is only valid if all participants sign it. Nobody can trick the wallet to sign a transaction where the user is loosing money. In other words, nobody can steal your money..</p>
```